miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/chat/ChatBridge.kt

```diff
@@ -104,5 +107,6 @@ internal class ChatBridge {
 
     internal companion object {
         const val ERR_SEND_MESSAGE = "Cannot send message:"
+        const val NULL_RESPONSE = "null"
```

Comment:
I think you can pass "null" directly or please move this into `ErrorBridgeMessage` in `MiniAppMessageBridge.kt`.